src/gui/properties/trackerlistwidget.cpp

```diff
@@ -82,6 +82,9 @@ TrackerListWidget::TrackerListWidget(PropertiesWidget *properties)
     for (int i = 0; i < COL_COUNT; ++i)
         if ((columnWidth(i) <= 0) && !isColumnHidden(i))
             resizeColumnToContents(i);
+    // Allow sort by column
+    for (int i = 0; i < COL_COUNT; ++i)
+        setSortingEnabled(i);
```

Comment:
```suggestion
    setSortingEnabled(true);
```
https://doc.qt.io/qt-6/qtreeview.html#sortingEnabled-prop
But unfortunately it's not that simple. Try changing tier for a tracker from the buttons on the right.

Reply:
It seemed too easy indeed. Well, nevermind then.

(I haven't even noticed the tier change buttons, even though I only use qBittorrent lol.)

Thank you for replying so fast.